charts/newrelic-infrastructure-beta/templates/ksm/deployment.yaml

```diff
@@ -0,0 +1,206 @@
+{{- if and (.Values.ksm.enabled) (.Values.enableLinux) }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}-ksm
+  {{- with .Values.ksm.annotations }}
+  annotations: {{ toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "newrelic.name" . }}-ksm
+      app.kubernetes.io/instance: {{ .Release.Name }}-ksm
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        app.kubernetes.io/name: {{ include "newrelic.name" . }}-ksm
+        app.kubernetes.io/instance: {{ .Release.Name }}-ksm
+        {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        mode: {{ include "newrelic.mode" . }}
+    spec:
+      {{- with .Values.images.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "newrelic.serviceAccountName" . }}
+      containers:
+        - name: forwarder
```

Comment:
Sometimes we put the `forwarder` container first, while others we put the integration container first. Mostly a nit but can hurt readability a bit. Could we unify this criteria?